### PR TITLE
Convert angular-easy-test to a UMD module

### LINKED ISSUE #18: Publish UMD module so the library can be used in AMD/CommonJS environments
i.e. https://github.com/umdjs/umd

### PR DESCRIPTION
**Pull request by elliottsj**
Resolves #18

Using webpack to build the [UMD](https://github.com/umdjs/umd) module; see README.md for usage.

This also changes the release process:
- the main distributable file is now `dist/easy-test.js`
- switched from grunt-bump to grunt-release since grunt-release provides a hook to build and commit `dist/easy-test.js` before releasing

### CODE REVIEW

File: lib/easy-test.js [hunk 1]
```diff
@@ -1,505 +1,506 @@
...
+ * });
+ */
+EasyTest.testController = function testScope(controller, spec) {
+  if (typeof controller === 'string') {
+    controller = EasyTest.getController(controller);
+  }
+  return EasyTest.looksLike(controller, spec);
+};
 
-  window.EasyTest = EasyTest;
```
Comment (SashaSirotkin) [inline lib/easy-test.js]: @elliottsj Once this change is in place, would projects need to use webpack (or what-have-you) to have access to easy test?

Comment (elliottsj) [inline lib/easy-test.js]: Nope; that's what [UMD](https://github.com/umdjs/umd) is for :grin:. UMD makes a single distributable module compatible with the three most popular module systems: CommonJS, AMD, and browser globals.

See https://github.com/elliottsj/angular-easy-test/blob/umd/dist/easy-test.js

Comment (SashaSirotkin) [inline lib/easy-test.js]: Where does it add to the browser globals?

Comment (elliottsj) [inline lib/easy-test.js]: The first argument to `webpackUniversalModuleDefinition` is `root`; ]the value for `root` is passed in as `this`](https://github.com/elliottsj/angular-easy-test/blob/umd/dist/easy-test.js#L10); in the browser's global scope, `this === window`, so the line
```js
root["easyTest"] = factory(root["angular"], root["angular"]["mock"]);
```
will assign `window.easyTest`.

aaand I just realized I should rename it to `EasyTest` :stuck_out_tongue:

Comment (elliottsj) [inline lib/easy-test.js]: Alright, now it's `EasyTest`

Comment (SashaSirotkin) [inline lib/easy-test.js]: Dope.

**Comment by SashaSirotkin** [comment]
As far I see, it looks fine to me. I would until @RickEyre's blessings before merger.

**Comment by RickEyre** [comment]
:whale: nice job. wepackifying is good.

**Pull request merged**